### PR TITLE
OE-377,set a redirect uri for cancel button

### PR DESCRIPTION
**Pull request by Zhaolian-Zhou**

### CODE REVIEW

File: protected/config/core/common.php [hunk 1]
```diff
@@ -321,7 +321,7 @@
                 'uri' => 'dashboard/cataract',
                 'position' => 4,
                 'userrule' => 'isSurgeon',
-                'restricted' => array('admin'),
+                'restricted' => array('admin,report'),
                 'options' => array('target' => '_blank'),),
```
Comment (apigram) [inline protected/config/core/common.php]: This change doesn't seem related to the fix. Can you explain its purpose?

Comment (Zhaolian-Zhou) [reply protected/config/core/common.php]: Oh, this related to the admin/cataract audit page, which seems request report role to show the page.

**Comment by Zhaolian-Zhou** [comment]
fixed

**Review by apigram — APPROVED** [review]
